tumblr-gif-search/index/_merge_matches_topk.yml

```diff
@@ -10,13 +10,13 @@ requests:
           traversal_paths: ['m']
       - !SortQL
         with:
-          reverse: False
+          reverse: false
           traversal_paths: ['m']
           field: 'score__value'
       - !SliceQL
         with:
           start: 0
-          end: $JINA_TOPK
+          end: 10
```

Comment:
Why we hardcode this instead of using the topk?

Reply:
because is updated by the `QuerySetReader` so is missleading